test/test_export.py

```diff
@@ -864,3 +865,59 @@ def func(a, b, c):
         dynamo_result = out_graph(*flat_input)
 
         self.assertTrue(torchdynamo.utils.same(real_result, dynamo_result))
+
+    def test_export_with_stack_trace(self):
+        inp = torch.tensor([0.1, 0.1])
+        linear = torch.nn.Linear(2, 2)
+
+        def func(x):
+            x = x + 1
+            y = x.t()
+            y = y.relu()
+            y = linear(y)
+            return y
+
+        exported = torchdynamo.export(func, inp, aten_graph=False)
+        out_graph = exported[0]
+
+        for node in out_graph.graph.nodes:
+            if node.op not in {"placeholder", "output"}:
+                self.assertTrue(node.stack_trace is not None)
+
+        torchdynamo.reset()
+
+        exported = torchdynamo.export(func, inp, aten_graph=True)
+        out_graph = exported[0]
+        for node in out_graph.graph.nodes:
+            if node.op == "call_function":
+                self.assertTrue(node.stack_trace is not None)
+
+    def test_export_compare_optimize_with_make_fx(self):
+        inp = torch.tensor([0.1, 0.1])
+        linear = torch.nn.Linear(2, 2)
+
+        def func(x):
+            x = x + 1
+            y = x.t()
+            y = y.relu()
+            y = linear(y)
+            return y
+
+        exported = torchdynamo.export(func, inp, aten_graph=True)
+        out_graph = exported[0]
+
+        torchdynamo.reset()
+
+        def compiler(gm, sample_inputs):
+            aten_gm = make_fx(gm)(*sample_inputs)
+
+            self.assertEqual(len(aten_gm.graph.nodes), len(out_graph.graph.nodes))
+            for node1, node2 in zip(aten_gm.graph.nodes, out_graph.graph.nodes):
+                self.assertEqual(node1.op, node2.op)
+                if node1.op == "call_function":
```

Comment:
Hmm what about the other nodes? Entire graph?

Reply:
Other nodes might have difference in names. 
For example, one graph would have input named arg0, another graph could have arg1_0... 
I think checking the target for call_function nodes should be sufficient...

Reply:
Hmm, can we at least ensure that the arg signature (number and type of args) and the output spec match across the graphs?

Reply:
Wouldn't evaluating the 2 graphs and asserting of getting same result be good enough?

Reply:
Awesome, thank you, and apologies for the insistence.